pkg/streaming/commands/kafkagateway_delete.go

```diff
@@ -72,12 +72,12 @@ func NewKafkaGatewayDeleteCommand(ctx context.Context, c *cli.Config) *cobra.Com
 		Use:   "delete",
 		Short: "delete kafka gateway(s)",
 		Long: strings.TrimSpace(`
-Delete one or more kafka gateways by name or all kafka gateways within a
+Delete one or more Kafka gateways by name or all Kafka gateways within a
 namespace.
 
-Deleting a kafka gateway will disrupt all processors consuming streams managed
+Deleting a Kafka gateway will disrupt all Processors consuming streams managed
```

Comment:
And a bunch of other places as well...

```suggestion
Deleting a Kafka gateway will disrupt all processors consuming streams managed
```

Reply:
why was this resolved?

Reply:
missed it, now fixed.